proxy/vm.go

```diff
@@ -150,9 +150,10 @@ func (vm *vm) ioHyperToClients() {
 
 		err = hyperstart.SendIoMessageWithConn(session.client, msg)
 		if err != nil {
-			fmt.Fprintf(os.Stderr,
-				"error writing I/O data to client: %v\n", err)
-			break
+			// When the shim is forcefully killed, it's possible we
+			// still have data to write. Ignore errors for that case.
+			vm.infof(1, "io", "error writing I/O data to client:", err)
+			continue
```

Comment:
The change looks like it might lead to an infinite loop - is it guaranteed that `vm.hyperHandler.ReadIoMessage()` will eventually kick us out of the loop when it returns an error?